{{ cookiecutter.project_slug }}/backend/requirements-docker.txt

```diff
@@ -0,0 +1,2 @@
+-c constraints.txt
+src/{{cookiecutter.python_package_name}}
```

Comment:
This one will be missing addons that are in requirements.txt.

To avoid duplicating them, I would keep the project Python package out of requirements.txt (so that we only need one), and install it after installing the requirements (as either an editable egg with test extras, or not, depending on the context)

Reply:
I'd prefer to tell people to add the additional production requirements to the setup.py of the package, instead of adding them to `requirements.txt`.
Opinions?

Reply:
@ericof I guess I'm okay with that. I see them as having 2 different purposes:
- setup.py defines the necessary dependencies for one Python package, ideally with permissive version ranges (allowing all versions that will actually work)
- requirements.txt defines the necessary dependencies for an entire virtualenv which probably has multiple Python packages, ideally pinned to specific versions so the build is repeatable

Having both can help catch errors if the virtualenv is installing a version that won't work with a specific package.

Reply:
I do want to make sure that the dependencies are in requirements.txt and installed in a separate Docker layer than installing the package that is created by the cookiecutter, because it makes building the Docker build much faster when the dependencies haven't changed. But it's not a big problem if the dependencies are also in setup.py, because they should already be installed from requirements.txt.

Reply:
be aware of the simple case that one does only need an addon and does not need any custom package.